### PR TITLE
BC-1408 - enabling school admin feature toggles

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/organisms/administration/PrivacySettings.vue [hunk 1]
```diff
@@ -245,7 +245,7 @@ export default {
 		lernStoreVisibility() {
 			return this.permissions.student
 				? this.permissions.student.LERNSTORE_VIEW
-				: false;
+				: true;
```
Comment (CeEv) [inline src/components/organisms/administration/PrivacySettings.vue:248]: For the PR, yes this is the needed changed. But in generally should we really have helper like lernStoreVisibility that include his own default state in code for the configuration settings?

Comment (Loki-Afro) [reply src/components/organisms/administration/PrivacySettings.vue:248]: if this is not the only place where this is checked yes, otherwise: unnecessary

**Review by CeEv — DISMISSED** [review]

File: src/components/organisms/administration/PrivacySettings.vue (hunk 1 above)
Comment (CeEv) [reply src/components/organisms/administration/PrivacySettings.vue:248]: The primäre problem is that the remove of the permission from students, should result in lernstore is not visible for students.
But that is also wrong implement im legacy client and must be fixed for it too.
Therefore approve from my side.

**Review by uidp — APPROVED** [review]

File: src/store/auth.ts [hunk 2]
```diff
@@ -178,6 +178,7 @@ export class Auth extends VuexModule {
 		return this.accesToken || false;
 	}
 
+	// TODO - why are we using toLowerCase() on permissions here?
```
Comment (CeEv) [inline src/store/auth.ts:181]: oh yes good questions... that make it hard to find something and work with it.

**Review by CeEv — APPROVED** [review]

**Pull request merged**